src/main/webapp/components/SearchBar.vue

```diff
@@ -43,17 +39,19 @@ module.exports = {
       }
       else /* Form validated */ {
         this.error = "";
-        this.queryDatabase(this.school, this.course)
-          .then(result => this.$emit('searchresult', result))
-          .catch(error => this.error = error);
+        this.queryDatabase(this.school, this.course);
```

Comment:
Should we handle the error that `queryDatabase()` can throw in some way?

Reply:
Sure I can add that